docker/owlbot/nodejs_mono_repo/Dockerfile

```diff
@@ -29,6 +29,9 @@ ENV PATH "$PATH:/usr/local/bin"
 ###################### Install git.
 RUN apt-get update && apt-get install -y git
 
+###################### Update image to latest
+RUN apt-get upgrade -y
```

Comment:
nit: consider combining with the previous `RUN` statement to limit the number of image layers